contrib/intel/jenkins/Jenkinsfile

```diff
@@ -409,14 +410,56 @@ pipeline {
                           sh """
                             env
                             (
-                                cd /opt${env.WORKSPACE}/py_scripts/contrib/intel/jenkins/
+                                cd /opt${env.WORKSPACE}/contrib/intel/jenkins/
                                 python3.7 runtests.py --prov=shm --device='ze'
                                 echo "ze-shm completed."
                             )
                           """
                         }
                     }
                 }
+                stage('dsa') {
+                    agent {node {label 'cst-pvc'}}
```

Comment:
Is the tests.py file changes missing from this? It looks like the --user_env option doesn't get used?